src/components/menu/menu.mdx

```diff
@@ -98,7 +98,7 @@ If you need to split out a submenu into a separate component, it must be done as
 
 In order to align text and icons within a submenu a `Box` component will need to be used to adjust the margin of the content.
 
-<Canvas of={MenuStories.SubmeuIconAndTextAlignment} />
+<Canvas of={MenuStories.SubmenuIconAndTextAlignment} />
```

Comment:
**issue**: would you be able to edit the name of the story as well? ([link to code](https://github.com/Sage/carbon/blob/master/src/components/menu/menu.stories.tsx#L371)). Currently, the docs crash as Storybook cannot find the story's code.